### PR TITLE
update service generator and projects

### PR DESCRIPTION
**Pull request by christianvaes**
update the microservice generator with all state of the art changes

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -41,11 +41,13 @@ module.exports = yeoman.generators.Base.extend({
       chalk.red('Maxxton Microservice generator\n\n') +
       yosay() + '\n\n' +
       '==============================\n' +
-      'Version: 1.0\n' +
+      'Version: 1.1\n' +
```
Comment (robin-maxxton) [inline app/index.js]: Maybe update the package.json so it also holds the right version (currently at version 0.0.0.)

File: app/templates/build.gradle [hunk 2]
```diff
@@ -3,8 +3,8 @@ buildscript {
     mavenCentral()
   }
   dependencies {
-    classpath("org.springframework.boot:spring-boot-gradle-plugin:1.2.5.RELEASE")
-    classpath("io.spring.gradle:dependency-management-plugin:0.5.2.RELEASE")
+    classpath("org.springframework.boot:spring-boot-gradle-plugin:1.3.0.RELEASE")
```
Comment (steven166) [inline app/templates/build.gradle]: Is Angel.SR4 compatible with spring boot 1.3.0?

Comment (christianvaes) [inline app/templates/build.gradle]: It works fine

Comment (robsonke) [inline app/templates/build.gradle]: It's not :). Spring cloud downgrades it to 1.2.6 again for now. Only the latest Brixton (kinda nightly) version works with 1.3.

**Pull request merged**